app/assets/main/react_components/routes/lifestyle_footprints/footprint_form_container/components/footprint_form/components/common/Preamble.jsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
```

Comment:
Only renaming from SignUpPreamble.jsx (looks like old file was deleted, it's git:s way of doing things)